web/src/lib/components/shared-components/create-share-link-modal/create-shared-link-modal.svelte

```diff
@@ -23,6 +25,7 @@
   export let onCreated: () => void = () => {};
 
   let sharedLink: string | null = null;
+  let showQRCode: boolean | null = false;
```

Comment:
When should `showQRCode` be null?

Reply:
you are right, boolean is never null. Ups, will fix it

Reply:
@danieldietzler thank you for your patience - i am new to this svelte/js world

Reply:
Happy to help :)